test/vector3.carp

```diff
@@ -14,7 +14,7 @@
   (assert-op test
              &(init 1.0 2.0 3.0) &(init 1.0 1.0 3.0)
              "/= operator works"
-             Vector3./=)
```

Comment:
Were these changes necessary now with the new env resolution, or is it just left over from testing something?

Reply:
Yes, it is necessary now because of the small lookup resolution changes. I think it’s actually what we want too. Before, because of the way we handled the module lookups, this line found the globally defined `/=` ( from Generics.carp) even though the author here was really trying to reference a function defined in a particular module—I think we want errors in this case otherwise there will be situations in which the resulting behavior is unexpected.